ide-common/src/main/kotlin/org/digma/intellij/plugin/analytics/BackendUtils.kt

```diff
@@ -1,11 +1,33 @@
 package org.digma.intellij.plugin.analytics
 
 import com.intellij.openapi.project.Project
+import org.digma.intellij.plugin.common.findActiveProject
 
 fun isCentralized(project: Project): Boolean {
     return BackendInfoHolder.getInstance(project).isCentralized()
 }
 
-fun getVersion(project: Project): String {
-    return BackendInfoHolder.getInstance(project).getAbout()?.applicationVersion ?: "unknown";
-}
\ No newline at end of file
+fun getBackendVersion(project: Project): String {
+    return BackendInfoHolder.getInstance(project).getAbout()?.applicationVersion ?: "unknown"
+}
+
+fun getBackendVersion(): String {
+    val project = findActiveProject()
+    return project?.let {
+        BackendInfoHolder.getInstance(it).getAbout()?.applicationVersion ?: "unknown"
```

Comment:
backend is responsible to return ApplicationVersion with "unknown" value

Reply:
this is the case where there is no connection to backend